src/docs/taiko.mdx

```diff
@@ -0,0 +1,165 @@
+---
+title: Taiko
+subtitle: Taiko is a decentralized ZK-Rollup. The protocol does not have a sequencer allowing anyone to become a proposer in a premissionless manner. Prover markets are implemented allowing anyone to be a prover and earn fees for proving L2 blocks.
+labels:
+    - ZK-Rollup
+    - EVM
+    - Testnet
+links:
+    website:
+        url: https://taiko.xyz/
+        name: taiko.xyz
+    docs:
+        url: https://taiko.xyz/docs
+        name: Docs
+    l2beat:
+        url: https://l2beat.com/scaling/projects/taiko
+        name: L2Beat
+    github:
+        url: https://github.com/taikoxyz
+        name: Github
+    twitter:
+        url: https://twitter.com/taikoxyz
+        name: taikoxyz
+    discord:
+        url: https://discord.com/invite/taikoxyz
+        name: Discord
+---
+
+<Section title="Overview">
+
+    Taiko is a decentralized, Type 1 ZK-Rollup. The protocol does not have a sequencer allowing anyone to become a proposer in a premissionless manner. Prover markets are implemented allowing anyone to be a prover and earn fees for proving L2 blocks.
+    
+    ###### Focus
+
+    - Full OPCODE EVM equivalence.
+
+
+    <Labels labels={['ZK-Rollup', 'EVM', 'Testnet']} title="Taiko" />
+
+</Section>
+
+<Section title="General">
+
+    <Parameter name="Block Time" value="On demand*" tooltip="The rate at which the rollup produces blocks. Blocks are produced on demand."/>
+
+    <Parameter name="Gas Limit" value="8.18 million" tooltip="The gas limit that can be consumed by an L2 block. 180 000 gas is reserved for the anchor transaction."/>
```

Comment:
From their docs it says that the gas limit is 6m, we will need to reconcile it.

https://taiko.xyz/docs/reference/differences-from-ethereum

Reply:
yes, however that's for testnet 3 and 4. We are at testnet 5 already (Jolnir). Reference from block explorer: https://explorer.jolnir.taiko.xyz/block/841551